source/internals/tasks.js

```diff
@@ -12,7 +12,7 @@ const tasks = {
   },
   wipeSystemiOSPodsCache: {
     name: 'wipe system iOS Pods cache',
-    command: 'cd ios & pod',
+    command: 'cd ios & bundle exec pod',
```

Comment:
Shouldn't it be like this?

```suggestion
    command: 'cd ios && bundle exec pod',
```

Reply:
Yes, you're right.